inc/auth.class.php

```diff
@@ -1155,12 +1135,10 @@ static function showSynchronizationForm(User $user) {
             case self::LDAP :
                //Look it the auth server still exists !
                // <- Bad idea : id not exists unable to change anything
-               $sql = "SELECT `name`
-                       FROM `glpi_authldaps`
-                       WHERE `id` = '" . $user->getField('auths_id') . "'
-                             AND `is_active` = 1";
-               $result = $DB->query($sql);
-               if ($DB->numrows($result) > 0) {
+               // SQL query
+               $result = $DB->request('glpi_authldaps', ['FIELDS' => 'name', ['id' => $user->getField('auths_id'), 'is_active' => 1]]);
```

Comment:
The doc is for "9.1" (stable version)
This recent changes are in master only
BTW, I was thinking to:
- sync DBinterator in 9.1 to make backport easier
- update documentation

@trasher what do you think ?